uptime_service_validation/coordinator/server.py

```diff
@@ -69,16 +73,58 @@ def setUpValidatorPods(time_intervals, logging, worker_image, worker_tag):
         keyspace = os.environ.get("AWS_KEYSPACE")
         start_timestamp = datetime_formatter(mini_batch[0]).replace(" ", "\ ")
         end_timestamp = datetime_formatter(mini_batch[1]).replace(" ", "\ ")
-
+        
+        # Unique names are needed for the configmaps
         configmap_name_suffix = str(
             "".join(random.choices(string.ascii_lowercase + string.digits, k=5))
         )
 
+        # The entrypoint script does the following
+        # Sources necessary credentials
+        # Resolves the domain name of the database host with nslookup
+        # Because for some unknown reason when resolving it throught the cqlsh binary the consistency of this working is lower than 50%
+        # If nslookup failed retries and if it succeded to get an ip address executes the command
+
         script_content = f"""#!/bin/bash
```

Comment:
Ideally, this script should live in the repository as a bash script and included with a read command in the variable

Reply:
Ideally, yes, but then we would need to move the logic of calculating the time intervals for the batches as well.

Every script has a different content, the timestamps are different for each, that is why I thought to use a formatted string and just include the variables `{keyspace} {start_timestamp} {end_timestamp}`

Do you think we should we go for the ideal solution?